libs/core/onboarding/components/onboarding-welcome.styled.tsx

```diff
@@ -3,49 +3,87 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import styled from "styled-components"
-import { fontWeight } from "Core/core/styles/theming/theme-getters"
+import styled, { css } from "styled-components"
+import {
+  backgroundColor,
+  borderColor,
+  fontWeight,
+  textColor,
+} from "Core/core/styles/theming/theme-getters"
 import ButtonComponent from "Core/__deprecated__/renderer/components/core/button/button.component"
+import Text from "Core/__deprecated__/renderer/components/core/text/text.component"
 
-export const OnboardingWrapper = styled.section`
+const baseContainerStyles = css`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+`
+
+export const Container = styled.section`
   display: grid;
-  grid-template-areas: "Header" "Main" "Footer";
+  grid-template-areas: "Content" "ButtonsContainer";
   grid-row-gap: 0;
-  grid-template-rows: 5.7rem 1fr auto;
-
-  header,
-  main,
-  footer {
-    text-align: center;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-  }
+  grid-template-rows: auto 13.2rem;
+  width: 100%;
+`
 
-  header {
-    grid-area: Header;
+export const Content = styled.div`
+  ${baseContainerStyles};
+  grid-area: Content;
+  justify-content: space-between;
+  background-color: ${backgroundColor("main")};
+`
 
-    p {
-      font-weight: ${fontWeight("light")};
-    }
-  }
+export const ContentTop = styled.div`
+  max-width: 107.2rem;
+  margin: 8rem auto 4rem auto;
+  text-align: center;
+`
 
-  main {
-    grid-area: Main;
-    justify-content: center;
-  }
+export const HeaderTitle = styled(Text)`
+  font-size: 4.8rem;
+  line-height: 4.8rem;
+  margin-bottom: 0.4rem;
+`
+export const SubheaderTitle = styled(Text)`
+  font-size: 2.2rem;
+  font-weight: ${fontWeight("light")};
+  color: ${textColor("info")};
+`
+
+export const ContentBottom = styled.div``
```

Comment:
this one can be removed and replaced by `main` tag in`OnboardingWelcome` component